### DISCUSSION TITLE
When will React Native support Android15

### DISCUSSION THREAD

**Discussion by nicevlhjfihtqngkldl** [react-native-community/discussions-and-proposals#798] [Feedback]
Hi, 
Many app markets now require developers to adapt their mobile apps to Android 15. May I ask when React Native will support Android 15?
Appreciate your help.

**Comment by cortinico** [comment]
Android 15 hasn't been released yet.
We'll follow up with platform updates as soon as it releases

**Comment by Nykolaj1979** [comment]
Hello team,
any updates in the mean time ?

**Comment by KPPriyadharshan** [comment]
any updates ??
